pkg/service/roommanager.go

```diff
@@ -433,13 +435,17 @@ func (r *RoomManager) StartSession(
 		_ = participant.Close(true, types.ParticipantCloseReasonJoinFailed, false)
 		return err
 	}
-	if r.config.PSRPC.Enabled {
-		if err := r.roomServer.RegisterAllParticipantTopics(rpc.FormatParticipantTopic(roomName, participant.Identity())); err != nil {
-			pLogger.Errorw("could not join register participant topic", err)
-			_ = participant.Close(true, types.ParticipantCloseReasonMessageBusFailed, false)
-			return err
-		}
+
```

Comment:
Should this still be conditioned on Enabled?

Reply:
iirc this was how we rolled out signal so in 6 months or whenever we make psrpc the default existing deployments will be forward compatible

Reply:
Sorry about the confusion. If this is not constrained by `Enabled`, is both legacy and PsRPC handling requests?

Reply:
yes

Reply:
got it, thank you @paulwe